symphony-bdk-core/src/main/java/com/symphony/bdk/core/service/message/MessageService.java

```diff
@@ -55,18 +41,14 @@
  */
 @Slf4j
 @API(status = API.Status.STABLE)
-public class MessageService {
+public class MessageService extends OboMessageService {
```

Comment:
I'm a bit afraid of inheritance bringing problems later as it introduces a lot of coupling, do we have use cases where you would need the message service to be used either as OBO or not OBO (i.e the need to have a common parent class/interface)

Have we looked at existing bots to see how they use OBO and if they mix OBO / non OBO usages?

Reply:
Composition instead of inheritance? We could consider using https://projectlombok.org/features/Delegate.html